packages/sdk-metrics/src/types.ts

```diff
@@ -24,18 +23,3 @@ export type CollectionOptions = CommonReaderOptions;
 export type ShutdownOptions = CommonReaderOptions;
 
 export type ForceFlushOptions = CommonReaderOptions;
-
-/**
- * @experimental
- *
- * This is intentionally not using the API's type as it's only available from @opentelemetry/api 1.9.0 and up.
- * In SDK 2.0 we'll be able to bump the minimum API version and remove this workaround.
- */
-export interface Gauge<
-  AttributesTypes extends MetricAttributes = MetricAttributes,
-> {
-  /**
-   * Records a measurement. Value of the measurement must not be negative.
-   */
-  record(value: number, attributes?: AttributesTypes, context?: Context): void;
-}
```

Comment:
Sorry for being unclear: We still need this type in `sdk-metrics` but we can remove the `@experimental` annotation.